packages/db/src/models/execution.ts

```diff
@@ -81,6 +83,32 @@ class ExecutionPgModel extends BasePgModel<PostgresExecution, PostgresExecutionR
       });
     return executions;
   }
+
+  /**
+   * Fetches multiple items from Postgres that contain a payload before a date
+   * this is included unused as a part of PR to ask, does this belong here?
```

Comment:
It's unused? In that case no 😄 

It's been a long time since we've had this debate about models BUT my opinion is that anything that's constrained to querying the datastore for a specific model is ok to have in the model vs. a lib. I'd say this belongs but you may find other opinions elsewhere 😄

Reply:
I've since removed this, I meant to before requesting PR, I decided the code was too specific to this script to belong in a standard model

Reply:
commit?